Issue-464-added a disclaimer for the requested information to be added (#903)
magento/merchdocs: src/magento/magento-account-share.md

   ![]({% link images/images/magento-account-shared-help-center.png %}){: .zoom}
   _Magento Help Center_

   {:.bs-callout-info}

>    After receiving shared access, the user needs to log in to [Magento.com](https://account.magento.com/customer/account/login), navigate to `Shared Access`, and click on the `Support` tab. This only needs to be done the first time. Doing this will ensure that the [Magento Help Center](https://support.magento.com/hc/en-us) is configured properly via the `SSO` call from Magento.com to Magento Help Center.

Reviewer: ```suggestion
   After receiving shared access, the user needs to log in to [Magento.com](https://account.magento.com/customer/account/login), navigate to _Shared Access_, and click the **Support** tab. This needs to be done only the first time. Doing this will ensure that the [Magento Help Center](https://support.magento.com/hc/en-us) is configured properly through the `SSO` call from Magento.com to the Magento Help Center.
```

Author: Hi @leslietilling, Thank you for the feedback, I have made the requested changes.
